src/index.js

```diff
@@ -61,13 +66,21 @@ module.exports = function (content, ...args) {
     this.cacheable();
   }
 
-  // let's only check `exports.locals` for keys to avoid getting keys from the sourcemap when it's enabled
-  // if we cannot find locals, then the module only contains global styles
-  const indexOfLocals = content.indexOf(".locals");
-  const cssModuleKeys =
-    indexOfLocals === -1
-      ? []
-      : getCssModuleKeys(content.substring(indexOfLocals));
+  let cssModuleKeys = [];
+
+  if (options.namedExport) {
+    const indexOfExports = content.indexOf("// Exports");
```

Comment:
I used `// Exports` as a marker since css-loader puts the comment.
https://github.com/webpack-contrib/css-loader/blob/e76ef2c0667b143bfa383efe379fa67142664319/src/utils.js#L563